specification/monitor/resource-manager/Microsoft.Insights/stable/2015-04-01/autoscale_API.json

```diff
@@ -803,6 +810,35 @@
           "type": "string"
         }
       }
+    },
+    "ScaleRuleMetricDimension": {
+      "required": [
+        "dimensionName",
+        "operator",
+        "values"
+      ],
+      "properties": {
+        "dimensionName": {
+          "type": "string",
+          "description": "Name of the dimension."
+        },
+        "operator": {
+          "type": "string",
+          "description": "the dimension operator. Only 'Equals' and 'NotEquals' are supported",
+          "enum": [
+            "Equals",
+            "NotEquals"
+          ]
```

Comment:
Might need a `x-ms-enum`

Reply:
Since the `values` is an array, `Equals` being "equal to any"? How about "NotEquals"? Might worth a more detailed description.